src/app/offers/offer-details/offer-details.component.html

```diff
@@ -217,70 +224,70 @@ <h1 class="app-basic-title">{{ states.project.name }}</h1>
             </div>
 
             <div class="offer-right second">
-            <div class="right__docs" *ngIf="states.project.documents.length !== 0"
-                 [class.add-padding]="states.project.news.length === 0">
-                <div class="docs__title app-page-subtitle">
-                    {{ 'offers.offer_details.subtitle_docs' | translate }}
-                </div>
-                <div *ngFor="let doc of states.project.documents">
-                    <div class="docs__links">
-                        <a href="{{ doc.link }}">
-                            <i class="fas fa-file-download"></i>
-                            <span>{{ doc.name }}</span>
-                        </a>
+                <div class="right__docs" *ngIf="states.project.documents.length !== 0"
+                     [class.add-padding]="states.project.news.length === 0">
+                    <div class="docs__title app-page-subtitle">
+                        {{ 'offers.offer_details.subtitle_docs' | translate }}
+                    </div>
+                    <div *ngFor="let doc of states.project.documents">
+                        <div class="docs__links">
+                            <a href="{{ doc.link }}">
+                                <i class="fas fa-file-download"></i>
+                                <span>{{ doc.name }}</span>
+                            </a>
+                        </div>
                     </div>
                 </div>
-            </div>
 
-            <div class="right__news" *ngIf="states.project.news.length !== 0">
-                <div class="news__title app-page-subtitle">
-                    {{ 'offers.offer_details.news.subtitle' | translate }}
-                </div>
+                <div class="right__news" *ngIf="states.project.news.length !== 0">
```

Comment:
Where is left news?

Reply:
This is a class that applies the styling rules for news in the right column on the screen. It is a wanna-be [BEM](http://getbem.com/) convention in naming things, but in this case, it was misused (used in a bad way), in my opinion. 

There is no news in the left column. if there is a news section in the left column, it would probably be named `left__news`.

There are plenty of concerns about naming classes (and naming in general). I've been looking around and testing `tailwindcss`, which is a popular CSS framework for CSS-utility classes meaning that you barely don't ever need to think about names and conventions, you just write utility classes inline. Take a look at [tailwindcss homepage](https://tailwindcss.com/)!